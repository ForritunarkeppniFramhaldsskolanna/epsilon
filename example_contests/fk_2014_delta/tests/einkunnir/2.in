3
FOR102 8.8
FOR203 9.2
FOR303 8.1
