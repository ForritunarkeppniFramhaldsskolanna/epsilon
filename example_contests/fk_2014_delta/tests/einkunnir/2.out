FOR 8.7
